rules/same-core-dependency-version/same-core-dependency-version.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+const request = require('sync-request');
+
+let packageJsonContents;
+
+module.exports = {
+  registerPackageJsonContents: fileContents => packageJsonContents = fileContents,
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'If a dependency is also specified in the kibi-internal repo, it must have the same version here',
+      category: 'Best Practices',
+      recommended: true,
+      url: 'https://github.com/sirensolutions/eslint-plugin-siren/blob/master/rules/same-core-dependency-version/same-core-dependency-version.md'
+    },
+    schema: [] // no options
+  },
+  create: context => {
+    if (!packageJsonContents || path.basename(context.getFilename()) !== 'package.json' || !process.env.GITHUB_TOKEN) {
```

Comment:
maybe add an early return if the repo is "kibi-internal" 
to avoid unnecessary CPU cycles

Reply:
Yeah, was going to disable in kibi-internal rather than hard code a rule